main.py

```
@@ -11,65 +11,80 @@
+        # Setup logging
+        setup_logging()
+
+        # Load environment variables
+        load_dotenv()
+
+        # Validate environment variables
+        database = os.getenv("MONGO_DATABASE")
+        temp_image_file = Path(os.getenv("TEMP_IMAGE_FILE", ""))
+
+        if not database:
+            raise ValueError("MONGO_DATABASE environment variable is required")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)